src/main/handlers/Utilities/Utilities.class.ts

```diff
@@ -129,6 +129,7 @@ class Utilities {
       if (
         (prerequisite.id === 'wsl' && stdOutput) ||
         (prerequisite.id === 'wslUbuntu' && stdOutput.includes(': Ubuntu')) ||
+        (prerequisite.id === 'wslUbuntuStore' && stdOutput) ||
```

Comment:
Can we update this check to check for something like `Running` in the stdout? What was your output of this command when Ubuntu was not installed from Windows Store?


![image](https://github.com/EtherealEngine/etherealengine-control-center/assets/10975502/19f99878-34b5-404d-97ba-983d418da36d)

Reply:
The thing is that `wslService` will only respond if it is downloaded from the Windows store, else it will drop error